rohit/Image-Search-Hooks/src/component/SearchBar.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+class SearchBar extends React.Component {
```

Comment:
Let's convert this into a functional component as well?

Reply:
okay we will do.